README.md

```diff
@@ -1,77 +1,136 @@
-# Template for `Application Examples`
+# Template for `Application Examples` on GitHub
 
 ## What is an application example
 
-An application example ist a stand alone project, which demonstrates the usage of one ore more features or libraries. This template is optimized for a creation of application examples for PLC applications (executable on a PLCSIM Advanced or real PLC S7-1500).
+An application example is a standalone project, which demonstrates the usage of one ore more features or libraries. This template is optimized for a creation of application examples for PLC applications (executable on a PLCSIM Advanced or real PLC S7-1500).
 
-## Create a project from this template
+## Create a project from an "apax-template"
```

Comment:
Don't agree. You this template is the start point. So keep "from this template"

Reply:
Agree ... although down the line the readme again is more precise and initially i wanted to have that section to be more "in genral".

Anyway I revert the changes for this part ... thx for the feedback

Reply:
I understand, but here it's a very specific instruction how to start exactly with this template.  Similar to some package descriptions. It's not a general instruction how to use templates.